package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "anagram",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
This seems like it might be a breaking change so we should probably bump the major version.  😉